src/Question.vue

```diff
@@ -3,53 +3,72 @@
         <div class="body-wrapper">
             <!-- Default slot is question body -->
             <slot></slot>
-            <div v-if="hasInput" class="textarea-container">
+            <div v-if="hasInputBool" class="textarea-container">
                 <textarea class="form-control question-input" rows="3" placeholder="write your answer here..."></textarea>
             </div>
         </div>
-        <accordion>
-            <panel v-show="hasHintSlot" header="Hint" expandable no-close>
-                <slot name="hint">
-                    No hint is available for this question.
-                </slot>
-            </panel>
-            <panel v-show="hasAnswerSlot" header="Answer" expandable no-close>
-                <slot name="answer"></slot>
-            </panel>
-        </accordion>
+        <panel v-show="hasHintSlot" header="Hint" expandable no-close>
+            <template v-if="isEmptyHint">
+                {{ defaultHint }}
```

Comment:
Since this is no longer controlled by slots, you can just dump the entire text here instead of using `{{defaultHint}}`.